examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>I think you're moving to a new section here, worth calling out the movement. </p><p> </p><p>"With all this machinery established, we can now clearly illustrate the difference between conditional and interventional distributions using the do-operator" </p><p> </p><p>Kind of a link to the opening paragraph and a signal to the reader that we're getting to the meat of the post. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Done</p>